build.sbt

```diff
@@ -25,20 +31,54 @@ def jsonDependencies(scalaVersion: String) = Seq(
   logback % Test
 ) ++ jacksons ++ specsBuild.map(_ % Test)
 
+// Common settings 
+import com.typesafe.sbt.SbtScalariform._
+import scalariform.formatter.preferences._
+
+val previousVersion = Some("2.6.0-M1") // first from this separate repo
```

Comment:
The goal is to check that, except the package change for `ValidationError`, most of the API remains compatible.

The filter about `ValidationError` has been added. Checking against M1 is equivalent (more simple for me), as the changes between 2.5.12 and 2.6.0-M1 are only about this package change.